Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'puma', '~> 3.0'
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 # gem 'rack-cors'
 gem 'active_model_serializers', '~> 0.10.0'
+gem 'simplecov', :require => false, :group => :test
 group :development, :test do
```

Comment:
I am unfamiliar with that notation so I didn't want to mess with it and ruin your work, we can alway refactor when after our function is 100%